package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "build": "webpack --config webpack.prod.js",
+    "heroku-postbuild": "webpack --config webpack.prod.js",
```

Comment:
Good. Please how can we test this?

Reply:
Since the heroku-postbuild and build scripts do the same thing, why not change it to:
`"heroku-postbuild": "npm run build"`

Reply:
True. Once heroku sees a `heroku-postbuild` script, it does not run the `build` script. So we can 'trick' it by running calling the build script in `heroku-postbuild` #smiles#

Reply:
> Good. Please how can we test this?

Will be tested when subsequent PRs are raised

Reply:
> Since the heroku-postbuild and build scripts do the same thing, why not change it to:
> `"heroku-postbuild": "npm run build"`

Very true. Will do that now

Reply:
@TheDrizzyWay `heroku-postbuild` script has been changed to `npm run build`